src/components/menu/menu-test.stories.tsx

```diff
@@ -455,5 +471,41 @@ export const MenuWithSubmenuCustomPadding = () => (
     ))}
   </>
 );
-
 MenuWithSubmenuCustomPadding.storyName = "Menu with submenu custom padding";
+
+export const WhenMenuItemsWrap = () => {
```

Comment:
**suggestion(non-blocking)**: what do you think about re-using one of the existing stories, by instructing Chromatic to capture it using a smaller viewport? This may help reduce the number of stories to maintain. Not essential though.

Reply:
I considered it but I want one of the items to have a very short title to achieve the snapshot I want so it was easier to make a new static story rather than make an existing one more complex